src/scripts.js

```diff
@@ -100,7 +125,8 @@ recipeGrid.addEventListener("mouseover", (event) => {
   }
 });
 
-closeRecipeButton.addEventListener("click", closeRecipe);
+closeRecipeButtons.forEach(btn => btn.addEventListener("click", closeRecipe));
+
```

Comment:
why the refactoring?

Reply:
I added a new close recipe button when the window is smaller, so they both get the event listener here